src/pages/platform/login.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useRef } from 'react'
+import { v4 } from 'uuid'
+
+const jwt = require('jsonwebtoken')
+import { GetServerSideProps } from 'next'
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  // TODO: verify token
+
+  const message = {
+    iss: process.env.PROVIDER_URL,
+    aud: ['editor'],
+    // TODO: no idea where this should be coming from
```

Comment:
It is supposed to be the id of the user that initiated the flow. `sub` is optional if it is an anonymous call. 
See https://www.imsglobal.org/spec/lti/v1p3 (search for 5.3.6 User Identity claims)